packages/edit-site/src/components/preferences-modal/enable-feature.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { ___unstablePreferencesModalBaseOption as BaseOption } from '@wordpress/interface';
+import { store as preferencesStore } from '@wordpress/preferences';
+
+export default function EnableFeature( props ) {
+	const { featureName, ...remainingProps } = props;
+	const isChecked = useSelect( ( select ) =>
+		select( preferencesStore ).get( 'core/edit-site', featureName )
```

Comment:
I'd maybe just coerce this into a bool to be on the safe side 😄 
```suggestion
		!! select( preferencesStore ).get( 'core/edit-site', featureName )
```